docs/usage.md

```diff
@@ -5,8 +5,9 @@
 
 # Usage
 
-The **FortiAI** solution pack offers a variety of functions through its **AI Assistant** to managing and automating tasks related to cybersecurity workflows. Following is an overview of the supported functions:
+The **FortiAI** solution pack offers a variety of functions for managing and automating tasks related to cybersecurity workflows. Following is an overview of the supported functions:
 
+- **Create Connector**: Create a connector using natural language prompts, review results, and download the associated connector code in the form of `info.json`, `operations.py` etc.
 - **Generate Playbook**: Generate, create a playbook or workflow, or playbook steps.
 - **Navigate Pages**: Navigate, open, or move to a specific page or module.
 - **Link Record**: Link or correlate a record with other relevant records. For example: correlating an indicator to an existing alert or incident.
```

Comment:
We also do Alert Triage as part of the SOC assistant

Reply:
SOC assistant, Playbook assistant, Connector assistant are all distinctive internally, There is only ONE assistant. AI Assistant/Chatbot.